Public/Src/Cache/ContentStore/Library/Stores/QuotaManagement/LegacyQuotaKeeper.cs

```diff
@@ -80,9 +79,12 @@ protected override async Task<BoolResult> StartupCoreAsync(OperationContext cont
             _reserveTask = Task.Run(() => Reserve(new Context(context)));
             _taskTracker = new BackgroundTaskTracker(Component, new Context(context));
 
-            // Start purging immediately on startup to clear out residual content in the cache
-            // over the cache quota
-            StartPurging();
+            if (PurgeAtStartup)

```

Comment:
>PurgeAtStartup [](start = 16, length = 14)

Can we emit some diagnostic when we choose to not purge at startup?

Reply:
It seems like we would have sufficient logging for when we do purge at startup.

---
In reply to: [273233305](https://github.com/Microsoft/BuildXL/pull/102#discussion_r273233305) [](ancestors = 273233305)

Reply:
That makes a lot of sense! Will add.

---
In reply to: [273233305](https://github.com/Microsoft/BuildXL/pull/102#discussion_r273233305) [](ancestors = 273233305)